browser/binance/static_values.h

```diff
@@ -19,6 +19,7 @@ const std::vector<std::string> kBinanceBlacklistRegions = {
   "UA",
   "CU",
   "IR",
+  "JP",
```

Comment:
We didn't need this before to control the widget display, as we used `onlyAnonWallet` from rewards. This just makes it so we don't have to check that anymore.

Reply:
ok